custom_components/mail_and_packages/const.py

```diff
@@ -982,10 +1082,12 @@
     "poczta_polska",
     "inpost_pl",
     "dpd_com_pl",
+    "dpd",
     "gls",
     "dhl_parcel_nl",
     "bonshaw_distribution_network",
     "purolator",
     "intelcom",
     "post_nl",
+    "post_de",
```

Comment:
```suggestion
```

We only put shippers here that have both `delivering` and `delivered` sensors.

Reply:
I thought so, got a bit confused because of Poczta post, as it is included without delivered